test/Microsoft.AspNetCore.Server.KestrelTests/HttpsConnectionFilterTests.cs

```diff
@@ -22,6 +23,13 @@ namespace Microsoft.AspNetCore.Server.KestrelTests
 {
     public class HttpsConnectionFilterTests
     {
+#if NET451
+        static HttpsConnectionFilterTests()
+        {
+            ServicePointManager.SecurityProtocol |= SecurityProtocolType.Tls12 | SecurityProtocolType.Tls11;
```

Comment:
Do all these tests fail trying to use SslProtocols.Tls on desktop if this isn't here?